Weight Converter/README.md

```diff
@@ -0,0 +1,19 @@
+# Weight Converter
+# Made by: Palak Singhania 
+
+# Description
+1. This is a simple weight converter project in which the user enters the weight in kilograms and it gets converted to 
+grams, pounds and ounces.
+2. It contains two files : index.html and main.css
+
+# screenshots
+a. When we don't enter any value or the value is less than or equal to zero then an alert appears.
+
+![](./screenshots/when_no_value_passed.png)
+
+b. When we enter a valid value then it gets converted on clicking convert button.
+
+![](./screenshots/smooth_functioning.png)
```

Comment:
Images not showing. @palaksinghania05

Reply:
But on my account, images are visible in readme.md. What to do?